weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ValueViewString.tsx

```diff
@@ -72,12 +72,35 @@ const PreserveWrapping = styled.div`
 `;
 PreserveWrapping.displayName = 'S.PreserveWrapping';
 
+// Regular expressions for common Markdown syntax
+// Note this is intentionally limited in scope to reduce false positives.
+const LIKELY_MARKDOWN_PATTERNS: RegExp[] = [
+  /```[\s\S]*```/, // Code block
+  /\[.+\]\(.+\)/, // Links [text](url)
+  /!\[.*\]\(.+\)/, // Images ![alt](url)
+];
+
+const isLikelyMarkdown = (value: string): boolean => {
+  return LIKELY_MARKDOWN_PATTERNS.some(pattern => pattern.test(value));
+};
+
+const getDefaultFormat = (value: string): string => {
```

Comment:
```suggestion
const getDefaultFormat = (value: string): 'Markdown' | 'Text' => {
```

Reply:
I think that plays nicer with the type suggestions, until we properly type it

Reply:
I tightened up the typing.